Felienne/hedy
# My profile UI was not aligned with other forms on site

**Description**

The "my profile" page is styled to better align with the UI of other forms on the website. We also hide the "classes" header when there are not classes. Accidentally solving 2 issues in one PR, but issue #1072 is really (really) small. Notice: The UI is still under construction, dependent on the decision on #1149.

**Fix for**

This fixed #994 and fixed #1072.  

**How to test**

Navigate to the "my profile" page to see the changes. For completeness the difference between the old and new version is shown below with screenshots:

**Old**
![Schermafbeelding 2021-11-03 om 10 13 10](https://user-images.githubusercontent.com/38155424/140034394-b4003dd9-be50-4007-9d36-71d17cb6c10c.png)

**New**
![Schermafbeelding 2021-11-03 om 10 12 45](https://user-images.githubusercontent.com/38155424/140034402-b174ae36-717f-48e6-883c-bfecbf83505b.png)

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
